Add option to show current temperature/humidifier on cards (#30429)
home-assistant/home-assistant.github.io: source/_dashboards/humidifier.markdown

  required: false
  description: Override the used theme for this card with any loaded theme. For more information about themes, see the [frontend documentation](/integrations/frontend/).
  type: string
show_current_as_primary:
  required: false

>   description: Show current humidity as primary information instead of target humidity. In that case, target humidity will be displayed as secondary information.

Reviewer: ```suggestion
  description: Show current humidity as primary information instead of target humidity. The target humidity will be displayed as secondary information.
```
